pkgs/by-name/ne/nexusmods-app/package.nix

```
@@ -189,7 +191,7 @@ buildDotnetModule (finalAttrs: {
     };
   };
 
-  passthru.updateScript = nix-update-script { };
+  passthru.updateScript = toString ./update.sh;
```

`toString` here was intentional, as in my testing `nix-instantiate --eval` would copy this to store if it was a path literal.

However this seems to be a quirk of `--raw`:

```sh
$ nix-instantiate --eval --raw --attr nexusmods-app.updateScript
/nix/store/56arpbw5l21axgxczhc9cx2q0ix0y0qj-update.sh
$ nix-instantiate --eval --attr nexusmods-app.updateScript
/home/matt/nixpkgs/nexus/pkgs/by-name/ne/nexusmods-app/update.sh
```

```suggestion
  passthru.updateScript = ./update.sh;
```